app.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'modules/create_people'
+require_relative 'modules/create_book'
+require_relative 'modules/create_rental'
+require_relative 'modules/list_books'
+require_relative 'modules/list_people'
+require_relative 'modules/list_rentals'
+
+class App
+  include CreatePeople
+  include CreateBook
+  include CreateRental
+  include ListBooks
+  include ListPeople
+  include ListRentals
+
+  def initialize()
+    @people = []
+    @books = []
+    @classrooms = []
+  end
+
+  def select_option(option)
```

Comment:
- [Optional] I think these are still the user's initial choice of options that should have been placed in the main.rb file as you have done with the others.